src/System.IO.FileSystem.DriveInfo/tests/DriveInfo.Unix.Tests.cs

```diff
@@ -41,27 +41,46 @@ public void TestGetDrives()
 
         [Fact]
         [PlatformSpecific(PlatformID.AnyUnix)]
-        public void TestInvalidDriveName()
+        public void PropertiesOfInvalidDrive()
         {
-            var invalidDrive = new DriveInfo("NonExistentDriveName");
-            Assert.Throws<DriveNotFoundException>(() => { var df = invalidDrive.DriveFormat; });
-            Assert.Throws<DriveNotFoundException>(() => { var size = invalidDrive.TotalFreeSpace; });
-            Assert.Throws<DriveNotFoundException>(() => { var size = invalidDrive.TotalSize; });
+            string invalidDriveName = "NonExistentDriveName";
+            var invalidDrive = new DriveInfo(invalidDriveName);
+
+            Assert.Throws<DriveNotFoundException>(() => { var invalid = invalidDrive.AvailableFreeSpace; });
+            Assert.Throws<DriveNotFoundException>(() => { var invalid = invalidDrive.DriveFormat; });
             Assert.Equal(DriveType.NoRootDirectory, invalidDrive.DriveType);
+            Assert.False(invalidDrive.IsReady);
+            Assert.Equal(invalidDriveName, invalidDrive.Name);
+            Assert.Equal(invalidDriveName, invalidDrive.ToString());
+            Assert.Equal(invalidDriveName, invalidDrive.RootDirectory.Name);
+            Assert.Throws<DriveNotFoundException>(() => { var invalid = invalidDrive.TotalFreeSpace; });
+            Assert.Throws<DriveNotFoundException>(() => { var invalid = invalidDrive.TotalSize; });
```

Comment:
We shouldn't need to store the results in locals, e.g. instead of:
```C#
Assert.Throws<DriveNotFoundException>(() => { var invalid = invalidDrive.TotalSize; });
```
you should be able to do:
```C#
Assert.Throws<DriveNotFoundException>(() => invalidDrive.TotalSize);
```